content/dart/concepts/variables/variables.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'Variables'
+Description: 'A variable is a location in computer memory used to store data.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Variables'
+  - 'Data Types'
+---
+
+A **`variable`** is a location in computer memory used to store references, usually for use in a program. Variables serve as symbolic names (identifiers) for values in the computer's memory. Variables allow programmers to manipulate and process data dynamically within their programs.
+
+## Syntax
+
+When declaring a variable in Dart, the type of variable goes first, followed by the name, and then the value:
+
+```pseudo
+type name = 'value';
+```
+
```

Comment:
You need to explain what `type` and `value` are and what different values they can take.
Something like this:
```
- `type`: It can be any data type like `Int`, `String` or can also take the value `var`, `const`
- `value`: This is what the variable `name` represents. It corresponds to the data type `type`.
```
Please feel free to write this in a better way, this is just to give an idea.